src/main/java/com/nike/cerberus/operation/cms/CreateCmsAsgOperation.java

```diff
@@ -122,7 +122,7 @@ public boolean isRunnable(CreateCmsClusterCommand command) {
         try {
             cloudFormationService.getStackId(region, Stack.LOAD_BALANCER.getFullName(environmentName));
             cloudFormationService.getStackId(region, Stack.SECURITY_GROUPS.getFullName(environmentName));
-            cloudFormationService.getStackId(configStore.getPrimaryRegion(), Stack.IAM_ROLES.getFullName(environmentName));
```

Comment:
Does it need both?

Reply:
No, it only needed the iam role previously in order to create the instance profile. Since instance profile creation happens in a separate command, only instance profile name is needed here. The iam role name is needed in the `create-instance-profile` command